request/helpers.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash');
+
+const buildPullRequestBody = (integration) => {
+  const {
+    apiServiceAccount,
+    authType,
+    clientId,
+    realm: realmName,
+    environments,
+    accountableEntity,
+    requester,
+    idpNames,
+    devValidRedirectUris,
+    testValidRedirectUris,
+    prodValidRedirectUris,
+  } = integration;
+
+  let body = `
+  #### Project Name: \`${_.startCase(clientId)}\`
```

Comment:
can we display `project name` from database?
and do we need this for `team account`?